lib/hydra/works/models/concerns/generic_file/contained_files.rb

```diff
@@ -10,6 +10,9 @@ module Hydra::Works::GenericFile::ContainedFiles
     directly_contains_one :original_file, through: :files, type: ::RDF::URI("http://pcdm.org/use#OriginalFile"), class_name: "Hydra::PCDM::File"
     directly_contains_one :thumbnail, through: :files, type: ::RDF::URI("http://pcdm.org/use#ThumbnailImage"), class_name: "Hydra::PCDM::File"
     directly_contains_one :extracted_text, through: :files, type: ::RDF::URI("http://pcdm.org/use#ExtractedText"), class_name: "Hydra::PCDM::File"
+    directly_contains_one :mp3, through: :files, type: ::RDF::URI("http://pcdm.org/use#ServiceFile"), class_name: "Hydra::PCDM::File"
```

Comment:
Sufia includes these 4 derivatives so I thought it would be good to pull down that behavior. The `directly_contains_one` asssociation allows you to call derivatives directly on the generic_file as follows: `generic_file.mp3`
https://github.com/projecthydra/sufia/blob/a064489d3cf4d228abf978d7126644e2c653aa5e/sufia-models/app/models/concerns/sufia/generic_file/derivatives.rb